.gitignore

```diff
@@ -11,3 +11,6 @@
 /tests/Application/yarn.lock
 
 .env.test
+
+# PHP-Storm
+.idea/
```

Comment:
Should be implemented in your local `.git/info/exclude`

Reply:
Yeah. No idea why this ended up in the commit. -.-